### DISCUSSION TITLE
[Support]  @midwayjs/cron 3.15.8有问题

### DISCUSSION THREAD

**Discussion by huihuicnXu** [midwayjs/midway#3785] [Q&A]
### Describe the problem（描述问题）

升级以后无法build
 "@types/cron": "2.4.0",
This is a stub types definition for @types/cron (https://github.com/kelektiv/node-cron#readme).

cron provides its own type definitions, so you don't need @types/cron installed!

### Midway Versions（Midway 版本）

升级以后无法build
"@types/cron": "2.4.0",
This is a stub types definition for @types/cron (https://github.com/kelektiv/node-cron#readme).

cron provides its own type definitions, so you don't need @types/cron installed!

**Comment by czy88840616** [comment]
https://github.com/midwayjs/midway/pull/3786

Trying to remove and runing test right now.
